styles/styles.css

```diff
@@ -665,7 +664,7 @@ button {
   -webkit-background-clip: text;
   /* stylelint-disable-line */
   -webkit-text-fill-color: transparent;
-  color: transparent;
+  color: var(--gradient-left-right);
```

Comment:
Why was this needed? Cross browser support?

Reply:
ooh, no I need to put that back to transparent. Thanks!